moirae/estimators/online/joint.py

```diff
@@ -56,24 +59,29 @@ def step(self, inputs: InputQuantities, measurements: OutputQuantities) -> \
         joint_estimate, output_predicted = self.filter.step(new_controls=refactored_inputs,
                                                             measurements=refactored_measurements)
 
-        return joint_estimate, output_predicted
+        return (joint_estimate.convert(conversion_operator=self.joint_model._hidden_conversion),
+                output_predicted.convert(conversion_operator=self.joint_model._output_conversion))
```

Comment:
Don't call private functions from another class. Make the interfaces that are required public.

Reply:
Fixed.